python/src/worker/tasks/background_expressed_genes.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, msg):
 
     def _format_result(self, result):
         # Return a list of formatted results.
-        return Result({"genes": result["genes"]})
+        return Result(result, error=self.error)
```

Comment:
if you want to include the errors, it is more consistent to pass them along exactly the same way as the result, no reason to add the to the class i.e.:
```
    def _format_result(self, result, error):
        # Return a list of formatted results.
        return Result(result, error=error)
```

Reply:
moved